spec/formatting.md

```diff
@@ -18,3 +18,17 @@ the local variable takes precedence.
 
 It is an error for a local variable definition to
 refer to a local variable that's defined after it in the message.
+
+## Bidirectional Isolation
+
+When a message part has a direction different from the message as a whole
+or when the part's direction is not known,
```

Comment:
```suggestion
When a message part has a direction different from the message as a whole,
```

@macchiati With reference to your comment https://github.com/unicode-org/message-format-wg/pull/315#discussion_r1021884363, would this be the sort of change you're looking for? Or have I perhaps understood your "minimum" incorrectly?

The intent here would be to reduce the complexity required of implementations, such that they only need to add isolates if they explicitly know that a message and a part have different directionality. For instance, if an instance doesn't have the capability to determine the directionality of a variable `$foo`, it would not be required to isolate it.

To be clear, my own preference would be to require isolation by default in that case, but to allow for an implementation to be configurable to act differently, isolating either less, more, or somehow differently than defined in the spec.